modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -21,6 +21,7 @@ After multi-factor authentication is enabled, users must verify their Anypoint P
 * Security key
 * Salesforce Authenticator
 
+Organizations created after April 30, 2022 require multi-factor authentication by default for all users. Organization administrators can xref:multi-factor-authentication.adoc#exempt-mfa-users[add individual user and service accounts to an exemption list] or xref:multi-factor-authentication.adoc#disable-org-mfa[disable MFA across the organization] (not recommended). Note that https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce users must have MFA enabled].
```

Comment:
I don't this option "disable MFA across the organization".
I'm not sure if "Salesforce users must have MFA enabled" is relevant to anypoint.